Bixal/move.mil
# Drupal core updates and react_tools module update

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

Updated packages with composer including drupal core. Also updated the react_tools modules.

- Updated drupal core to 8.6.14 and other composer packages
- Updated react_tools packages

## Testing

To verify the changes proposed in this pull request…

1. move to project root
2. check drupal core version
3. move to web/modules/custom/react_tools/tools 
4. check that no npm modules have vulnerabilities
